db/models.py

```diff
@@ -14,3 +14,39 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
```

Comment:
You are overusing f-string here. The f-string is not necessary when you are returning only one variable. It would be better to simply return the variable itself. This is a code efficiency issue.